### ISSUE TITLE
Slow drift after movement

### ISSUE THREAD

**Issue by gsf600y** [mircearoata/SatisfactoryPakUtilityMod#24]
When flying there is a small amount of drift once you stop pressing movement keys.  It's very slow, but makes placing objects problematic, as you slowly drift and end up placing objects it a slightly different location to intended.

For some reason, ALT-TAB'ing out of the game twice slows the drifting to a stop.

**Comment by Strome76** [comment]
I have this issue to.. Although Alt-TAB:ing does nothing for me.

**Comment by gsf600y** [comment]
Yep, that's **exactly** the same for me, although, as previously mentioned, ALT-TAB back to the desktop, then again back to the game seems to halt the movement for me.

**Comment by Strome76** [comment]
Using alt-tab does not help unless I use true fullscreen. I normally use windowed fullscreen.
